### ISSUE TITLE
Error using M-x documentation-describe-bindings

### ISSUE THREAD

**Issue by rpx99** [lem-project/lem#861]
With Lem 2.1 windows release I get a crash when using `M-x documentation-describe-bindings`

![lem_crash_bindings](https://github.com/lem-project/lem/assets/70011162/a8ebc85d-2454-47e6-84ba-2eaec3c1cee1)

The command `M-x describe-bindings` works.

**Comment by vindarel** [comment]
This appears to be an issue with a *release*, which doesn't know about an ASDF Lem system.

```lisp
(asdf:component-children (asdf:find-component :lem "commands"))
                                                ^^^ => NIL in a release
```

If using Deploy, we could do `(when (deploy:deployed-p) …)`.  I think we need to run this collector function at compile/build time, save the list in a variable, which will be available in the binary release.

**Comment by cxxxr** [comment]
Hmm, I see.

> If using Deploy, we could do (when (deploy:deployed-p) …). I think we need to run this collector function at compile/build time, save the list in a variable, which will be available in the binary release.

Since Lem have a variable called `lem-core::*deployed*`, maybe we could use that.

**Comment by Sasanidas** [comment]
(ping @rpx99 ) Is this issue still happening? (with the latest release)

**Comment by rpx99** [comment]
Yes, I tested it with LEM 2.1 release.

**Comment by Sasanidas** [comment]
I guess this is the same problem as with other window https://github.com/lem-project/lem/issues/1174#issuecomment-1842933266 (basically that the Lem source code is not visible for quicklisp).

**Comment by Zoybean** [comment]
This is happening for me on fedora with current main (be73d0df) (plus jfaz1/sdl2-fix (92a7810a) so I can invoke the command in question)
